### DISCUSSION TITLE
How to install Apollo in Windows?

### DISCUSSION THREAD

**Discussion by Galaxy-228** [GMOD/Apollo#2651] [Q&A]
Hi:
I want use Apollo in my laptop(Windows System).
Thanks!

**Comment by garrettjstevens** [comment]
To run the Apollo server in Windows, you'll probably have to use Docker (see [here](https://genomearchitect.readthedocs.io/en/latest/Docker.html)). If you're connecting to a server hosted elsewhere, though, any modern browser on any operating system should work.

**Reply by Galaxy-228** [reply]
Thanks so much !!
